web/src/shared/components/SeqInfo.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react'
```

Comment:
There's a lot going on in this component, and the render logic can be hard to follow at times. I think it would be great to do a pair programming session where we can refactor and break it down into simpler components together. We can discuss some design strategies and their trade-offs. What do you think? @danielreti

Reply:
Let's do this together in Melbourne